pages/generate-test-cases.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Card, Grid, Text } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { useRecoilValue } from 'recoil';
+import TestCaseEditor from '../components/TestCaseEditor';
+import { measureBundleState } from '../state/atoms/measureBundle';
+import { measurementPeriodStringState } from '../state/selectors/measurementPeriodString';
+import { Edit } from 'tabler-icons-react';
+
+export default function TestCaseEditorPage() {
+  const measurementPeriodString = useRecoilValue(measurementPeriodStringState);
+  const measureBundle = useRecoilValue(measureBundleState);
+  return (
+    <>
+      <Grid>
```

Comment:
We should probably just yoink this and replace the `AppHeader` component? And if the card is only rendered when the measurement period is populated, that's nice? what are your thoughts?